src/main/java/uk/gov/hmcts/reform/civil/callback/OrderCallbackHandler.java

```diff
@@ -28,7 +28,9 @@ protected OrderCallbackHandler(DashboardApiClient dashboardApiClient, DashboardN
 
     protected boolean isEligibleForReconsideration(CaseData caseData) {
         return caseData.isSmallClaim()
-            && (caseData.getTotalClaimAmount().compareTo(BigDecimal.valueOf(1000)) <= 0)
+            && caseData.getTotalClaimAmount().compareTo(BigDecimal.valueOf(1000)) <= 0
+            && (isNull(caseData.getDecisionOnRequestReconsiderationOptions())
+            || !"CREATE_SDO".equals(caseData.getDecisionOnRequestReconsiderationOptions().name()))
```

Comment:
!DecisionOnRequestReconsiderationOptions.CREATE_SDO.equals(caseData.getDecisionOnRequestReconsiderationOptions().name()))

I think is more elegant than string....